articles/interface.html

```diff
@@ -517,7 +517,7 @@ <h3>Reflection</h3>
 <p>
 This article will not explain the functionalities provided by
 <a href="https://golang.org/pkg/reflect/">the <code>reflect</code> standard package</a>.
-Please read <!---a href="reflection.html"-->Reflections In Go<!---/a-->
+Please read <a href="reflection.html">Reflections In Go</a>

```

Comment:
change "Reflections In Go" to "reflections in Go"?
So that it is consistent with other article titles in content?